schematic/schemas/data_model_jsonld.py

```diff
@@ -0,0 +1,284 @@
+        '''
+        for rels in data_model_relationships.values():
+            if rels['jsonld_key'] in template.keys() and not template[rels['jsonld_key']]:
+                if 'jsonld_default' in rels.keys():
+                    template[rels['jsonld_key']] = rels['jsonld_default']
+                else:
+                    del template[rels['jsonld_key']]
+        return template
+
+    def strip_context(self, context_value: str) -> tuple[str]:
+        """Strip contexts from str entry.
+        Args:
+            context_value, str: string from which to strip context from
+        Returns:
+            context, str: the original context
+            v, str: value separated from context
+        """
+        if ':' in context_value:
+            context, v = context_value.split(':')
+        elif '@' in context_value:
+            context, v = context_value.split('@')
+        return context, v
+
+    def reorder_template_entries(self, template:dict) -> dict:
+        '''In JSONLD some classes or property keys have list values. We want to make sure these lists are ordered according to the order supplied by the user.
+        This will look specically in lists and reorder those.
+        Args:
+            template, dict: JSONLD template for a single entry, keys specified in property and class templates.
+        Returns:
+            template, dict: list entries re-ordered to match user supplied order.
+        Note:
+            User order only matters for nodes that are also attributes
+        '''
+        template_label = template['rdfs:label']
+
+        for jsonld_key, entry in template.items():
+            # Make sure dealing with an edge relationship:
+            is_edge = ['True' for k, v in self.rel_dict.items() if v['jsonld_key']==jsonld_key if v['edge_rel'] == True]
+            
+            #if the entry is of type list and theres more than one value in the list attempt to reorder
+            if is_edge and isinstance(entry, list) and len(entry)>1:
+                # Get edge key from data_model_relationships using the jsonld_key:
+                key, edge_key = [(k, v['edge_key']) for k, v in self.rel_dict.items() if jsonld_key == v['jsonld_key']][0]
+                
+                # Order edges
+                sorted_edges = self.DME.get_ordered_entry(key=key, source_node_label=template_label)
+                edge_weights_dict={edge:i for i, edge in enumerate(sorted_edges)}
+                ordered_edges = [0]*len(edge_weights_dict.keys())
+                for k,v in edge_weights_dict.items():
+                    ordered_edges[v] = {'@id': 'bts:' + k}
+                
+                # Throw an error if ordered_edges does not get fully filled as expected.
+                if 0 in ordered_edges:
+                    logger.error("There was an issue getting values to match order specified in the data model, please submit a help request.")
```

Comment:
I am seeing an error related to `logger.error` in my vs code. I think you might forget to import `logger`